src/VRDriver.cpp

```diff
@@ -167,6 +173,16 @@ void SlimeVRDriver::VRDriver::RunFrame()
         hmdPosition->set_qw((float) q.w);
 
         sendBridgeMessage(*message, *this);
+
+        vr::ETrackedPropertyError err;
+        if (vr::VRProperties()->GetBoolProperty(vr::VRProperties()->TrackedDeviceToPropertyContainer(0), vr::Prop_DeviceProvidesBatteryStatus_Bool, &err) == true) {
+            messages::Battery* hmdBattery = google::protobuf::Arena::CreateMessage<messages::Battery>(&arena);
+            message->set_allocated_battery(hmdBattery);
+            hmdBattery->set_tracker_id(0);
+            hmdBattery->set_battery_level(vr::VRProperties()->GetFloatProperty(vr::VRProperties()->TrackedDeviceToPropertyContainer(0), vr::Prop_DeviceBatteryPercentage_Float, &err) * 100);
+            hmdBattery->set_is_charging(vr::VRProperties()->GetBoolProperty(vr::VRProperties()->TrackedDeviceToPropertyContainer(0), vr::Prop_DeviceIsCharging_Bool, &err));
+            sendBridgeMessage(*message, *this);
```

Comment:
I found this unexpected, but if SlimeVR Server uses/relays the HMD battery level, then that's fine.

Reply:
It's mostly just a little cosmetic thing. I thought about doing the same for other devices like controllers and Vive trackers, but I think that's something for the Feeder App.

![SlimeVR 5_22_2023 6_56_51 PM](https://github.com/SlimeVR/SlimeVR-OpenVR-Driver/assets/21149527/02fe163c-6be0-4697-8708-81a2aa544833)

Reply:
If you could make an issue on the feeder app repo to remind me to send these messages, that would be appreciated <3